proto/finschia/or/rollup/v1/rollup.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+package finschia.or.rollup.v1;
+
+import "cosmos_proto/cosmos.proto";
+import "google/protobuf/any.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/Finschia/finschia-sdk/x/or/rollup/types";
+
+message Rollup {
+  string rollupName = 1;
+  string creator     = 2;
+  Sequencers permissionedAddresses = 3 [(gogoproto.nullable) = false];
+  uint64 maxSequencers = 4;
```

Comment:
If multiple sequencers can be registered for one rollup, they will be used.
Now it is fixed at 1.

https://github.com/Finschia/finschia-sdk/pull/1034/files#diff-29fc36437b7c2481fedf4fcd616f2f1aa9b47a35ed9a0cc07341f45a8feb266aR68

Reply:
I think the value specified in MsgCreateRollup will be this value, but is it okay to set this value in request?
I think it is better to be decided inside the rollup module rather than being decided by the user issuing the request.
https://github.com/Finschia/finschia-sdk/pull/1034/files/dcd6d654e7eb97df207c822c20abb4dceafacc32#diff-9dfff786e5d6f6a569ad5afac902871268457bb8ecd3b9c99bc60be8d771bf89R24